server/app/controllers/applicant/ApplicantInformationController.java

```diff
@@ -61,11 +61,11 @@ public ApplicantInformationController(
   }
 
   /**
-   * Sets the applicant's preferred language based on their browser session, then redirects them
+   * Sets the user's preferred language based on their browser session, then redirects them
```

Comment:
(non-blocking) I'd keep this as applicant since it's an applicant controller.

Reply:
Changed.